src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -66,6 +66,8 @@ public string ClientVersion
 
         public bool IsBrokerEnabled { get; internal set; }
 
+        public bool IsSsoPolicyEnabled { get; internal set; }

```

Comment:
Naming: `SSO` is quite overloaded term. Maybe `IsWebviewSsoPolicyEnabled` ? And a comment that this affects only PCA.

Reply:
This policy may not only scope to webview.

We can use IsPlatformAutomaticAppSignInPolicyEnabled. which is coming from the public API:
[https://learn.microsoft.com/en-us/uwp/api/windows.system.profile.platformautomaticappsigninpolicy?view=winrt-insider](https://learn.microsoft.com/en-us/uwp/api/windows.system.profile.platformautomaticappsigninpolicy?view=winrt-insider)